easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-2023.03.0+386-foss-2022b-Java-11-R-4.2.2.eb

```diff
@@ -0,0 +1,100 @@
+easyblock = 'CMakeNinja'
+
+name = 'RStudio-Server'
+version = '2023.03.0+386'
+versionsuffix = '-Java-%(javaver)s-R-%(rver)s'
+local_git_rev = '7872775ebddc40635780ca1ed238934c3345c5de'
+
+homepage = 'https://www.rstudio.com/'
+description = """This is the RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:
+  rserver --server-daemonize=0 --www-port=8787
+
+If you need a database config one can be created with:
+  MYTMP=`mktemp -d` && echo -e "provider=sqlite\\ndirectory=${MYTMP}/sqlite" > "${MYTMP}/db.conf"
+and then used with:
+  rserver ... --database-config-file="${MYTMP}/db.conf"
+"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = ['https://github.com/rstudio/rstudio/archive']
+sources = ['v%(version)s.tar.gz']
```

Comment:
Are we missing this patch? https://github.com/ComputeCanada/easybuild-easyconfigs/blob/computecanada-main/easybuild/easyconfigs/r/RStudio-Server/rstudio-1.2.1335.patch